mypy/modulefinder.py

```diff
@@ -75,7 +75,8 @@ def error_message_templates(self, daemon: bool) -> tuple[str, list[str]]:
         doc_link = "See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports"
         if self is ModuleNotFoundReason.NOT_FOUND:
             msg = 'Cannot find implementation or library stub for module named "{module}"'
-            notes = [doc_link]
+            pip_install_suggestion = "The following command may help: pip install {module}"
```

Comment:
Currently shows
```
The following command may help: pip install installer.records
```
but should look like
```
The following command may help: pip install installer
```

Reply:
Ah, I'm not too sure about how to fix this.

Reply:
If we split on `.`, and always return the first element, would that work in all cases?

Reply:
I don't know enough Python libraries to think of any counter-examples.